libs/core/src/scss/interaction-state/_focus.scss

```diff
@@ -36,6 +37,30 @@
   }
 }
 
+// -----------------------------------------------------------------------------
+// In some cases a focus ring is not desirable, e.g., when buttons are stacked
+// (see Action Sheet).
+// As an alternative change the state layer opacity - similar to hover.
+// -----------------------------------------------------------------------------
+@mixin apply-focus-visible-without-ring(
```

Comment:
Could we consider something like apply-focus-visible-layer or -background (even though not entirely technically true) to make it a bit more descriptive?

Reply:
Absolutely! 🙂